[2024-07-01 21:28:31,285: INFO: main: >>>>>> stage Data Ingestion stages started <<<<<<]
[2024-07-01 21:28:31,285: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-01 21:28:31,285: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-01 21:28:31,285: INFO: common: created directory at: artifacts]
[2024-07-01 21:28:31,285: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-01 21:28:45,940: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 11616915
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "adf745abc03891fe493c3be264ec012691fe3fa21d861f35a27edbe6d86a76b1"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 4E8D:F514A:2E0467:3CF41D:6682C1C8
Accept-Ranges: bytes
Date: Mon, 01 Jul 2024 15:58:32 GMT
Via: 1.1 varnish
X-Served-By: cache-bom4728-BOM
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1719849512.354049,VS0,VE263
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 687a4f2f9b0715b2f037ed3baaf18ead415b7f75
Expires: Mon, 01 Jul 2024 16:03:32 GMT
Source-Age: 0

]
[2024-07-01 21:28:46,288: INFO: data_ingestion: Zip file extracted to artifacts/data_ingestion]
[2024-07-01 21:28:46,288: INFO: main: >>>>>> stage Data Ingestion stages completed <<<<<<

x================x]
[2024-07-01 21:28:46,288: INFO: main: >>>>>> stage Prepare Base Model stage started <<<<<<]
[2024-07-01 21:28:46,288: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-01 21:28:46,303: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-01 21:28:46,303: INFO: common: created directory at: artifacts]
[2024-07-01 21:28:46,303: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-07-01 21:28:46,746: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-07-01 21:28:46,920: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-07-01 21:28:47,046: INFO: main: >>>>>> stage Prepare Base Model stage completed <<<<<<

x================x]
[2024-07-01 21:28:47,046: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-07-01 21:28:47,062: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-01 21:28:47,062: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-01 21:28:47,062: INFO: common: created directory at: artifacts]
[2024-07-01 21:28:47,062: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2024-07-01 21:28:47,062: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2024-07-01 21:28:47,062: INFO: common: created directory at: artifacts\training]
[2024-07-01 21:28:47,220: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-01 21:33:56,449: INFO: main: >>>>>> stage Data Ingestion stages started <<<<<<]
[2024-07-01 21:33:56,464: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-01 21:33:56,464: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-01 21:33:56,464: INFO: common: created directory at: artifacts]
[2024-07-01 21:33:56,464: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-01 21:34:10,798: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 11616915
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "adf745abc03891fe493c3be264ec012691fe3fa21d861f35a27edbe6d86a76b1"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 4E8D:F514A:2E0467:3CF41D:6682C1C8
Accept-Ranges: bytes
Date: Mon, 01 Jul 2024 16:03:57 GMT
Via: 1.1 varnish
X-Served-By: cache-bom4732-BOM
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1719849837.244760,VS0,VE259
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: bb5dcda2c956dd8f98ec848aaab87dc8ac8ee43e
Expires: Mon, 01 Jul 2024 16:08:57 GMT
Source-Age: 0

]
[2024-07-01 21:34:11,123: INFO: data_ingestion: Zip file extracted to artifacts/data_ingestion]
[2024-07-01 21:34:11,123: INFO: main: >>>>>> stage Data Ingestion stages completed <<<<<<

x================x]
[2024-07-01 21:34:11,123: INFO: main: >>>>>> stage Prepare Base Model stage started <<<<<<]
[2024-07-01 21:34:11,123: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-01 21:34:11,123: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-01 21:34:11,123: INFO: common: created directory at: artifacts]
[2024-07-01 21:34:11,131: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-07-01 21:34:11,797: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-07-01 21:34:11,939: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-07-01 21:34:12,056: INFO: main: >>>>>> stage Prepare Base Model stage completed <<<<<<

x================x]
[2024-07-01 21:34:12,056: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-07-01 21:34:12,056: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-01 21:34:12,066: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-01 21:34:12,066: INFO: common: created directory at: artifacts]
[2024-07-01 21:34:12,066: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2024-07-01 21:34:12,066: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2024-07-01 21:34:12,066: INFO: common: created directory at: artifacts\training]
[2024-07-01 21:34:12,365: WARNING: module_wrapper: From C:\Python312\Lib\site-packages\keras\src\backend\common\global_state.py:82: The name tf.reset_default_graph is deprecated. Please use tf.compat.v1.reset_default_graph instead.
]
[2024-07-01 21:34:12,681: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-01 21:42:40,866: INFO: main: >>>>>> stage Training completed <<<<<<

x================x]
[2024-07-02 09:27:32,872: INFO: main: >>>>>> stage Data Ingestion stages started <<<<<<]
[2024-07-02 09:27:32,872: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-02 09:27:32,872: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-02 09:27:32,872: INFO: common: created directory at: artifacts]
[2024-07-02 09:27:32,872: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-02 09:27:32,982: ERROR: main: <urlopen error [Errno 11001] getaddrinfo failed>]
Traceback (most recent call last):
  File "C:\Python312\Lib\urllib\request.py", line 1344, in do_open
    h.request(req.get_method(), req.selector, req.data, headers,
  File "C:\Python312\Lib\http\client.py", line 1331, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "C:\Python312\Lib\http\client.py", line 1377, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "C:\Python312\Lib\http\client.py", line 1326, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "C:\Python312\Lib\http\client.py", line 1085, in _send_output
    self.send(msg)
  File "C:\Python312\Lib\http\client.py", line 1029, in send
    self.connect()
  File "C:\Python312\Lib\http\client.py", line 1465, in connect
    super().connect()
  File "C:\Python312\Lib\http\client.py", line 995, in connect
    self.sock = self._create_connection(
                ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\socket.py", line 828, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\socket.py", line 963, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
socket.gaierror: [Errno 11001] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Asus\OneDrive\Documents\Data Science\Coccidiosis_Chicken_Diseases_classification\main.py", line 11, in <module>
    obj.main()
  File "c:\users\asus\onedrive\documents\data science\coccidiosis_chicken_diseases_classification\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 15, in main
    data_ingestion.download_file()
  File "c:\users\asus\onedrive\documents\data science\coccidiosis_chicken_diseases_classification\src\cnnClassifier\components\data_ingestion.py", line 13, in download_file
    filename, headers = request.urlretrieve(
                        ^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\urllib\request.py", line 240, in urlretrieve
    with contextlib.closing(urlopen(url, data)) as fp:
                            ^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\urllib\request.py", line 215, in urlopen
    return opener.open(url, data, timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\urllib\request.py", line 515, in open
    response = self._open(req, data)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\urllib\request.py", line 532, in _open
    result = self._call_chain(self.handle_open, protocol, protocol +
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\urllib\request.py", line 492, in _call_chain
    result = func(*args)
             ^^^^^^^^^^^
  File "C:\Python312\Lib\urllib\request.py", line 1392, in https_open
    return self.do_open(http.client.HTTPSConnection, req,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\urllib\request.py", line 1347, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [Errno 11001] getaddrinfo failed>
[2024-07-02 09:28:33,158: INFO: main: >>>>>> stage Data Ingestion stages started <<<<<<]
[2024-07-02 09:28:33,158: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-02 09:28:33,158: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-02 09:28:33,158: INFO: common: created directory at: artifacts]
[2024-07-02 09:28:33,158: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-02 09:28:44,544: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 11616915
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "adf745abc03891fe493c3be264ec012691fe3fa21d861f35a27edbe6d86a76b1"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 51BE:1FEEBD:1A9002:23EEAF:66837AE9
Accept-Ranges: bytes
Date: Tue, 02 Jul 2024 03:58:34 GMT
Via: 1.1 varnish
X-Served-By: cache-bom4736-BOM
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1719892713.369717,VS0,VE941
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 41ab15d474ec93db210b19a394a434254ba8b4d0
Expires: Tue, 02 Jul 2024 04:03:34 GMT
Source-Age: 1

]
[2024-07-02 09:28:44,876: INFO: data_ingestion: Zip file extracted to artifacts/data_ingestion]
[2024-07-02 09:28:44,876: INFO: main: >>>>>> stage Data Ingestion stages completed <<<<<<

x================x]
[2024-07-02 09:28:44,876: INFO: main: >>>>>> stage Prepare Base Model stage started <<<<<<]
[2024-07-02 09:28:44,876: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-02 09:28:44,892: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-02 09:28:44,892: INFO: common: created directory at: artifacts]
[2024-07-02 09:28:44,892: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-07-02 09:28:45,508: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-07-02 09:28:45,749: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-07-02 09:28:45,876: INFO: main: >>>>>> stage Prepare Base Model stage completed <<<<<<

x================x]
[2024-07-02 09:28:45,876: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-07-02 09:28:45,876: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-02 09:28:45,876: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-02 09:28:45,876: INFO: common: created directory at: artifacts]
[2024-07-02 09:28:45,876: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2024-07-02 09:28:45,876: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2024-07-02 09:28:45,876: INFO: common: created directory at: artifacts\training]
[2024-07-02 09:28:46,323: WARNING: module_wrapper: From C:\Python312\Lib\site-packages\keras\src\backend\common\global_state.py:82: The name tf.reset_default_graph is deprecated. Please use tf.compat.v1.reset_default_graph instead.
]
[2024-07-02 09:28:46,688: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-02 09:29:12,731: INFO: main: >>>>>> stage Data Ingestion stages started <<<<<<]
[2024-07-02 09:29:12,731: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-02 09:29:12,731: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-02 09:29:12,731: INFO: common: created directory at: artifacts]
[2024-07-02 09:29:12,731: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-02 09:29:20,611: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 11616915
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "adf745abc03891fe493c3be264ec012691fe3fa21d861f35a27edbe6d86a76b1"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 51BE:1FEEBD:1A9002:23EEAF:66837AE9
Accept-Ranges: bytes
Date: Tue, 02 Jul 2024 03:59:12 GMT
Via: 1.1 varnish
X-Served-By: cache-bom4721-BOM
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1719892752.337619,VS0,VE1
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 15d76e4f96f7227a07a0ec6c11155854134dcb06
Expires: Tue, 02 Jul 2024 04:04:12 GMT
Source-Age: 39

]
[2024-07-02 09:29:20,990: INFO: data_ingestion: Zip file extracted to artifacts/data_ingestion]
[2024-07-02 09:29:20,990: INFO: main: >>>>>> stage Data Ingestion stages completed <<<<<<

x================x]
[2024-07-02 09:29:21,006: INFO: main: >>>>>> stage Prepare Base Model stage started <<<<<<]
[2024-07-02 09:29:21,006: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-02 09:29:21,006: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-02 09:29:21,006: INFO: common: created directory at: artifacts]
[2024-07-02 09:29:21,006: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-07-02 09:29:21,511: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-07-02 09:29:21,780: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-07-02 09:29:22,201: INFO: main: >>>>>> stage Prepare Base Model stage completed <<<<<<

x================x]
[2024-07-02 09:29:22,201: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-07-02 09:29:22,216: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-02 09:29:22,216: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-02 09:29:22,238: INFO: common: created directory at: artifacts]
[2024-07-02 09:29:22,238: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2024-07-02 09:29:22,238: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2024-07-02 09:29:22,249: INFO: common: created directory at: artifacts\training]
[2024-07-02 09:29:22,701: WARNING: module_wrapper: From C:\Python312\Lib\site-packages\keras\src\backend\common\global_state.py:82: The name tf.reset_default_graph is deprecated. Please use tf.compat.v1.reset_default_graph instead.
]
[2024-07-02 09:29:23,047: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-02 09:37:22,009: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-07-02 09:37:22,083: INFO: main: >>>>>> stage Training completed <<<<<<

x================x]
[2024-07-02 10:27:07,871: INFO: main: >>>>>> stage Data Ingestion stages started <<<<<<]
[2024-07-02 10:27:07,871: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-02 10:27:07,871: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-02 10:27:07,871: INFO: common: created directory at: artifacts]
[2024-07-02 10:27:07,887: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-02 10:27:13,376: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 11616915
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "adf745abc03891fe493c3be264ec012691fe3fa21d861f35a27edbe6d86a76b1"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 51BE:1FEEBD:1A9002:23EEAF:66837AE9
Accept-Ranges: bytes
Date: Tue, 02 Jul 2024 04:57:08 GMT
Via: 1.1 varnish
X-Served-By: cache-bom4745-BOM
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1719896228.901785,VS0,VE321
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: ad44d4aa404c2ba76c250c57d095bbae8c979cdb
Expires: Tue, 02 Jul 2024 05:02:08 GMT
Source-Age: 0

]
[2024-07-02 10:27:13,676: INFO: data_ingestion: Zip file extracted to artifacts/data_ingestion]
[2024-07-02 10:27:13,676: INFO: main: >>>>>> stage Data Ingestion stages completed <<<<<<

x================x]
[2024-07-02 10:27:13,676: INFO: main: >>>>>> stage Prepare Base Model stage started <<<<<<]
[2024-07-02 10:27:13,676: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-02 10:27:13,676: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-02 10:27:13,676: INFO: common: created directory at: artifacts]
[2024-07-02 10:27:13,676: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-07-02 10:27:14,198: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-07-02 10:27:14,467: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-07-02 10:27:14,561: INFO: main: >>>>>> stage Prepare Base Model stage completed <<<<<<

x================x]
[2024-07-02 10:27:14,561: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-07-02 10:27:14,561: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-02 10:27:14,561: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-02 10:27:14,561: INFO: common: created directory at: artifacts]
[2024-07-02 10:27:14,577: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2024-07-02 10:27:14,577: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2024-07-02 10:27:14,577: INFO: common: created directory at: artifacts\training]
[2024-07-02 10:27:14,891: WARNING: module_wrapper: From C:\Python312\Lib\site-packages\keras\src\backend\common\global_state.py:82: The name tf.reset_default_graph is deprecated. Please use tf.compat.v1.reset_default_graph instead.
]
[2024-07-02 10:27:15,199: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-02 10:35:50,618: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-07-02 10:35:50,697: INFO: main: >>>>>> stage Training completed <<<<<<

x================x]
[2024-07-02 10:35:50,697: INFO: main: >>>>>> stage Evaluation started <<<<<<]
[2024-07-02 10:35:50,697: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-02 10:35:50,697: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-02 10:35:50,697: INFO: common: created directory at: artifacts]
[2024-07-02 10:35:50,869: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-02 10:36:11,631: INFO: common: scores.json saved successfully]
[2024-07-02 10:36:11,631: INFO: main: >>>>>> stage Evaluation completed <<<<<<

x================x]
[2024-07-02 11:02:30,424: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stages started <<<<<<]
[2024-07-02 11:02:30,426: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-02 11:02:30,428: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-02 11:02:30,428: INFO: common: created directory at: artifacts]
[2024-07-02 11:02:30,430: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-02 11:02:46,572: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 11616915
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "adf745abc03891fe493c3be264ec012691fe3fa21d861f35a27edbe6d86a76b1"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 51BE:1FEEBD:1A9002:23EEAF:66837AE9
Accept-Ranges: bytes
Date: Tue, 02 Jul 2024 05:32:30 GMT
Via: 1.1 varnish
X-Served-By: cache-bom4738-BOM
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1719898351.554614,VS0,VE248
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 0b7fa1e68f648c5593b94b1c0b0fd4f12d665ab5
Expires: Tue, 02 Jul 2024 05:37:30 GMT
Source-Age: 0

]
[2024-07-02 11:02:46,831: INFO: data_ingestion: Zip file extracted to artifacts/data_ingestion]
[2024-07-02 11:02:46,831: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stages completed <<<<<<

x================x]
[2024-07-02 11:09:31,562: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stages started <<<<<<]
[2024-07-02 11:09:31,562: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-02 11:09:31,578: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-02 11:09:31,578: INFO: common: created directory at: artifacts]
[2024-07-02 11:09:31,578: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-02 11:09:38,638: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 11616915
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "adf745abc03891fe493c3be264ec012691fe3fa21d861f35a27edbe6d86a76b1"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 51BE:1FEEBD:1A9002:23EEAF:66837AE9
Accept-Ranges: bytes
Date: Tue, 02 Jul 2024 05:39:31 GMT
Via: 1.1 varnish
X-Served-By: cache-bom4725-BOM
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1719898772.579717,VS0,VE314
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: ffbb442f462d716278940c9087b101d7c6567fd3
Expires: Tue, 02 Jul 2024 05:44:31 GMT
Source-Age: 0

]
[2024-07-02 11:09:38,891: INFO: data_ingestion: Zip file extracted to artifacts/data_ingestion]
[2024-07-02 11:09:38,891: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stages completed <<<<<<

x================x]
[2024-07-02 11:09:46,627: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare Base Model stage started <<<<<<]
[2024-07-02 11:09:46,627: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-02 11:09:46,627: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-02 11:09:46,627: INFO: common: created directory at: artifacts]
[2024-07-02 11:09:46,627: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-07-02 11:09:46,976: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-07-02 11:09:47,103: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-07-02 11:09:47,181: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare Base Model stage completed <<<<<<

x================x]
[2024-07-02 11:09:50,891: INFO: stage_03_training: >>>>>> stage Training started <<<<<<]
[2024-07-02 11:09:50,891: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-02 11:09:50,891: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-02 11:09:50,891: INFO: common: created directory at: artifacts]
[2024-07-02 11:09:50,891: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2024-07-02 11:09:50,891: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2024-07-02 11:09:50,891: INFO: common: created directory at: artifacts\training]
[2024-07-02 11:09:51,191: WARNING: module_wrapper: From C:\Python312\Lib\site-packages\keras\src\backend\common\global_state.py:82: The name tf.reset_default_graph is deprecated. Please use tf.compat.v1.reset_default_graph instead.
]
[2024-07-02 11:09:51,564: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-02 11:16:33,876: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-07-02 11:16:33,956: INFO: stage_03_training: >>>>>> stage Training completed <<<<<<

x================x]
[2024-07-02 11:16:37,772: INFO: stage_04_evaluation: >>>>>> stage Evaluation started <<<<<<]
[2024-07-02 11:16:37,772: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-02 11:16:37,777: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-02 11:16:37,777: INFO: common: created directory at: artifacts]
[2024-07-02 11:16:37,994: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-02 11:16:55,176: INFO: common: scores.json saved successfully]
[2024-07-02 11:16:55,184: INFO: stage_04_evaluation: >>>>>> stage Evaluation completed <<<<<<

x================x]
[2024-07-02 12:09:21,824: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.98.192:8080]
[2024-07-02 12:09:21,824: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-07-02 12:09:29,570: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:09:29] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:09:29,582: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:09:29] "GET / HTTP/1.1" 200 -]
[2024-07-02 12:09:29,735: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:09:29] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:09:29,746: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:09:29] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:11:51,200: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-02 12:11:51,750: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:11:51] "POST /predict HTTP/1.1" 200 -]
[2024-07-02 12:12:01,001: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-02 12:12:01,367: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:12:01] "POST /predict HTTP/1.1" 200 -]
[2024-07-02 12:13:24,581: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-02 12:13:24,946: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:13:24] "POST /predict HTTP/1.1" 200 -]
[2024-07-02 12:13:50,146: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:13:50] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:13:50,146: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:13:50] "GET / HTTP/1.1" 200 -]
[2024-07-02 12:13:50,146: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:13:50] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:13:50,221: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:13:50] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:13:54,910: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:13:54] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:13:54,916: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:13:54] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:14:00,436: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:14:00] "GET /train HTTP/1.1" 200 -]
[2024-07-02 12:14:00,498: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:14:00] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:14:30,798: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:14:30] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:14:30,802: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:14:30] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:14:30,805: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:14:30] "[31m[1mGET /predict HTTP/1.1[0m" 405 -]
[2024-07-02 12:14:38,295: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:14:38] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:14:38,298: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:14:38] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:14:38,302: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:14:38] "GET / HTTP/1.1" 200 -]
[2024-07-02 12:14:38,383: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:14:38] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:14:40,327: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-02 12:14:40,393: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "C:\Python312\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\flask_cors\extension.py", line 178, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\flask_cors\decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
                         ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Asus\OneDrive\Documents\Data Science\Coccidiosis_Chicken_Diseases_classification\app.py", line 34, in predictRoute
    result = clApp.classifier.predict()
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\users\asus\onedrive\documents\data science\coccidiosis_chicken_diseases_classification\src\cnnClassifier\pipeline\predict.py", line 14, in predict
    test_image = image.load_img(imagename, target_size=(224, 224))
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\keras\src\utils\image_utils.py", line 236, in load_img
    img = pil_image.open(io.BytesIO(f.read()))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\PIL\Image.py", line 3339, in open
    raise UnidentifiedImageError(msg)
PIL.UnidentifiedImageError: cannot identify image file <_io.BytesIO object at 0x000001EC4DEED030>
[2024-07-02 12:14:40,411: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:14:40] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-07-02 12:14:46,619: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:14:46] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:14:46,619: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:14:46] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:14:47,896: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:14:47] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:14:47,899: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:14:47] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:14:47,938: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:14:47] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:14:53,110: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-02 12:14:53,492: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:14:53] "POST /predict HTTP/1.1" 200 -]
[2024-07-02 12:15:01,526: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-02 12:15:01,876: WARNING: polymorphic_function: 5 out of the last 5 calls to <function TensorFlowTrainer.make_predict_function.<locals>.one_step_on_data_distributed at 0x000001EC53D19800> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-07-02 12:15:01,876: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:15:01] "POST /predict HTTP/1.1" 200 -]
[2024-07-02 12:15:55,740: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:15:55] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:15:55,745: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:15:55] "GET / HTTP/1.1" 200 -]
[2024-07-02 12:15:55,753: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:15:55] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:15:55,869: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:15:55] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:15:57,252: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:15:57] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:15:57,266: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:15:57] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:15:57,283: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:15:57] "GET / HTTP/1.1" 200 -]
[2024-07-02 12:15:57,394: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:15:57] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:16:11,219: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.98.192:8080]
[2024-07-02 12:16:11,219: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-07-02 12:16:14,786: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:16:14] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:16:14,796: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:16:14] "GET / HTTP/1.1" 200 -]
[2024-07-02 12:16:14,797: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:16:14] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:16:14,930: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:16:14] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-07-02 12:16:21,109: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-02 12:16:21,474: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:16:21] "POST /predict HTTP/1.1" 200 -]
[2024-07-02 12:16:27,013: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-02 12:16:27,355: INFO: _internal: 127.0.0.1 - - [02/Jul/2024 12:16:27] "POST /predict HTTP/1.1" 200 -]
